chaos-experiments/kubernetes/scripts/corruptSnapshot.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash 
+set -exo pipefail
+
+
+partition=$1
+
+# Find the latest snapshot directory
+snapshotsPath=data/raft-partition/partitions/$partition/snapshots
+snapshotDir=$(ls -la $snapshotsPath | grep -o -E "([0-9]+-[0-9]+-[0-9]+)")
+
+# Remove a random *.sst file
```

Comment:
Does this really corrupt a snapshot? What I observed when I was writing qa test is that rocksdb cannot detect that one .sst file is deleted.

Reply:
I thought that I saw that the snapshot can't be opened in the log. But I can verify. Probably we should anyway add a check that the old leader is again leader.

Reply:
@deepthidevaki I tried it again and as I said I see that the follower is not able to become leader.


The following is logged on the follower:
```
 2020-04-28T08:46:49.860Z Available snapshots: [SnapshotImpl{path=/usr/local/zeebe/data/raft-partition/partitions/3/snapshots/302999-1-1588063492612, compactionBound=302999}] 
E 2020-04-28T08:46:49.902Z Failed to open snapshot 'SnapshotImpl{path=/usr/local/zeebe/data/raft-partition/partitions/3/snapshots/302999-1-1588063492612, compactionBound=302999}'. No snapshots available to recover from. Manual action is required. 
I 2020-04-28T08:46:49.902Z Unexpected failures occurred when installing leader services, stepping down 
```

Reply:
If it works then it is good :+1: